weave-js/src/components/PagePanelComponents/Home/Browse3.tsx

```diff
@@ -411,9 +414,12 @@ const Browse3ProjectRoot: FC<{
         <Route path={`${projectRoot}/calls/:itemName`}>
           <CallPageBinding />
         </Route>
-        <Route path={`${projectRoot}/:tab(evaluations|traces|calls)`}>
+        <Route path={`${projectRoot}/:tab(traces|calls)`}>
           <CallsPageBinding />
         </Route>
+        <Route path={`${projectRoot}/evaluations/:tab(evals|monitors)`}>
```

Comment:
what happens when someone navigates to the old `/evaluations` route? should we default to evals?